Microsoft.DotNet.Interactive.Formatting/Formatter.cs

```diff
@@ -173,6 +173,12 @@ public static void ResetToDefault()
 
             _mimeTypesByType.Clear();
             ConfigureDefaultPlainTextFormattersForSpecialTypes();
+
+            Formatter<LaTeX>.Register((laTex, writer) =>
```

Comment:
We have default registrations in other places, e.g. `DefaultPlainTextFormatterSet`. Let's try to keep mime type-specific logic out of `Formatter`.